github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -603,6 +603,80 @@ periodics:
           memory: 8Gi
       securityContext:
         privileged: true
+- annotations:
+    testgrid-dashboards: kubevirt-periodics
+    testgrid-days-of-results: "60"
+  cluster: ibm-prow-jobs
+  cron: 35 6 * * *
+  decorate: true
+  decoration_config:
+    grace_period: 5m0s
+    timeout: 4h0m0s
+  extra_refs:
+  - base_ref: main
+    org: kubevirt
+    repo: kubevirt
+  - base_ref: main
+    org: kubevirt
+    repo: project-infra
+  labels:
+    preset-bazel-cache: "true"
+    preset-bazel-unnested: "true"
+    preset-github-credentials: "true"
+    preset-kubevirtci-quay-credential: "true"
+    preset-pgp-bot-key: "true"
+  max_concurrency: 1
+  name: periodic-kubevirt-e2e-k8s-1.26-sev
+  reporter_config:
+    slack:
+      job_states_to_report: []
+  spec:
+    containers:
+    - command:
+      - /usr/local/bin/runner.sh
+      - /bin/sh
+      - -c
+      - |
+        # install yq
+        curl -Lo ./yq https://github.com/mikefarah/yq/releases/download/3.4.1/yq_linux_amd64
+        chmod +x ./yq && mv ./yq /usr/local/bin/yq
+
+        # install sshuttle and kubectl
+        dnf -y install sshuttle kubernetes-client
+        # get kubeconfig and ssh key
+        source ../project-infra/hack/manage-secrets.sh
+        decrypt_secrets
+        extract_secret 'amdKubeConfig' /kubeconfig
+        extract_secret 'amdSEVClusterKey' /ssh-key && chmod 400 /ssh-key
+
+        # login quay.io
+        cat "$QUAY_PASSWORD" | docker login --username $(cat "$QUAY_USER") --password-stdin=true quay.io
+
+        # Set up tunnel
+        sshuttle -D -r kubevirt@165.204.53.121 10.216.91.126/32 --ssh-cmd 'ssh -o StrictHostKeyChecking=no -i /ssh-key -p 30026'
+
+        # run test
+        make cluster-sync && FUNC_TEST_LABEL_FILTER='--label-filter=(SEV)' make functest && make cluster-clean || make cluster-clean
```

Comment:
I don't see the PR that introduces the label into the KubeVirt code base, can you give me a pointer where [decorators.go](https://github.com/kubevirt/kubevirt/blob/main/tests/decorators/decorators.go) is modified?

Reply:
AH, found it: https://github.com/kubevirt/kubevirt/pull/7197/files#diff-2be6ff8bee7a7b41764cd824e3e7afbc1ae3facd2893bdeb60f7ac978303a387R18

Reply:
I think its added here - https://github.com/kubevirt/kubevirt/pull/7197/commits/6f0c9fa9e340ba1973b959034953d8fa6dd78247